core/agent_loop.py

```
@@ -682,189 +683,115 @@ class ReflectionResult(BaseModel):
-    
-                # 显示进度
-    
+
+                # 显示进度    
                 cli.display_progress(step + 1, self.max_steps, f"步骤 {step + 1}")
     
     
     
-                # 每轮开始前重新加载上下文（动态）
-    
+                # 每轮开始前重新加载上下文（动态）   
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)